universal-application-tool-0.0.1/app/services/applicant/RepeatedEntity.java

```diff
@@ -1,24 +1,86 @@
 package services.applicant;
 
 import com.google.auto.value.AutoValue;
-import services.question.types.QuestionDefinition;
+import com.google.common.collect.ImmutableList;
+import java.util.Optional;
+import services.Path;
+import services.question.types.EnumeratorQuestionDefinition;
 
 /** A repeated entity represents one of the applicant's answers to an enumerator question. */
 @AutoValue
 public abstract class RepeatedEntity {
 
-  public static RepeatedEntity create(
-      QuestionDefinition enumeratorQuestionDefinition, String entityName) {
+  /**
+   * Create all the non-nested repeated entities associated with the enumerator question, with no
+   * parent.
+   */
+  public static ImmutableList<RepeatedEntity> createRepeatedEntities(
+      EnumeratorQuestionDefinition enumeratorQuestionDefinition, ApplicantData applicantData) {
+    return RepeatedEntity.createRepeatedEntities(
+        Optional.empty(), enumeratorQuestionDefinition, applicantData);
+  }
+  /**
+   * Create all the nested repeated entities associated with the enumerator question, with this
```

Comment:
question: Trying to figure out "nested" vs "non-nested". This is just like one level deeper?

Do we just need the one above so that we can start with the "initial" case of having no repeated entity parents?

Reply:
The `static` method is for creating non-nested repeated entities.

The instanced method is for creating nested repeated entities.